cli/cmd/resources.go

```diff
@@ -0,0 +1,84 @@
+package cmd
+
+import (
+	backend "github.com/chanzuckerberg/happy/cli/pkg/backend/aws"
+	"github.com/chanzuckerberg/happy/cli/pkg/cmd"
+	"github.com/chanzuckerberg/happy/cli/pkg/config"
+	"github.com/chanzuckerberg/happy/cli/pkg/orchestrator"
+	"github.com/chanzuckerberg/happy/cli/pkg/output"
+	stackservice "github.com/chanzuckerberg/happy/cli/pkg/stack_mgr"
+	"github.com/chanzuckerberg/happy/cli/pkg/workspace_repo"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	rootCmd.AddCommand(resourcesCmd)
+	config.ConfigureCmdWithBootstrapConfig(resourcesCmd)
+}
+
+var resourcesCmd = &cobra.Command{
+	Use:          "resources",
+	Short:        "Get stack resources",
+	Long:         "Get stack resources in environment '{env}'",
+	SilenceUsage: true,
+	PreRunE:      cmd.Validate(cobra.ExactArgs(1)),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		ctx := cmd.Context()
+		stackName := args[0]
+
+		bootstrapConfig, err := config.NewBootstrapConfig(cmd)
+		if err != nil {
+			return err
+		}
+		happyConfig, err := config.NewHappyConfig(bootstrapConfig)
+		if err != nil {
+			return err
+		}
```

Comment:
There's a helper for this:
```suggestion
		happyConfig, err := config.GetHappyConfigForCmd(cmd)
		if err != nil {
			return err
		}
```